src/router/index.js

```diff
@@ -18,6 +18,7 @@ import AdminBroker from '@/components/AdminBroker'
 import AdminSampling from '@/components/AdminSampling'
 import UserTasks from '@/components/UserTasks'
 import ConfigInsights from '@/components/ConfigInsights'
+import ResourceDistribution from '@/components/ResourceDistribution'
```

Comment:
Can you add a screenshot of this from browser ?

Reply:
not stacked:
![image](https://user-images.githubusercontent.com/10009508/101794040-beb04e00-3b06-11eb-9f17-68355b9e74e9.png)

stacked:
![image](https://user-images.githubusercontent.com/10009508/101794229-f0c1b000-3b06-11eb-8ec4-70f15bb28b67.png)

obviously we can work on the colors and on the size, especially on the stacked one.
I have added the `replicas` options in the last commit

Reply:
looks good, thank you.